examples/js/controls/OrbitControls.js

```diff
@@ -5,6 +5,8 @@
 //    Zoom - middle mouse, or mousewheel / touch: two-finger spread or squish
 //    Pan - right mouse, or left mouse + ctrl/meta/shiftKey, or arrow keys / touch: two-finger move
 
+const clock = new THREE.Clock();
+let _dampingFactor
```

Comment:
This would add a `clock` and `_dampingFactor` to `window`.

Reply:
Also it should be `var` instead of `let`/ `const`.